plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/NoImplementationWidget.java

```diff
@@ -11,45 +11,29 @@
 package org.eclipse.che.ide.ext.java.client.navigation.openimplementation;
```

Comment:
I still don't understand why we need a special widget to show "no implementations."

Reply:
@tsmaeder what do you propose to use instead? sorry if you already said this somewhere and i missed it

Reply:
If a PopupWidget has no elements, it shows the string returned from "getEmtpyMessage()". I don't understand why that is not what we want.

Reply:
Yes, but we have a "ImplementationWidget" and a "NoImplementationWidget". Why?